### PR TITLE
[Sonic Frontiers] Disable Quick Cyloop Target Override

### PR DESCRIPTION
**Pull request by wAABBsif**
We couldn't really come up with a better name for this code.

### CODE REVIEW

**Comment by hyperbx** [comment]
Can you explain what this code does and where it'd be useful? It's not very clear.

File: HedgeModManager/Resources/Codesv2/SonicFrontiers.hmm [hunk 1]
```diff
@@ -3070,6 +3070,21 @@ Code "Homing Attack Trick Animations" in "Animation" by "Hyper & Ahremic" does "
...
 
+Patch "Disable Quick Cyloop Target Override" in "Gameplay" by "WasifBoomz & The Duck Dealer" does "Disables the lock-on change following a Quick Cyloop."
+{
+    WriteProtected<byte>
+    (
+        /* 0x14088D334 */
+        ScanSignature
+        (
+            "\x76\x12\x8B\x46\x28\x48\x8D\x55\x97\x48\x8B\xCB\x89\x45\x97\xE8\xCC\xCC\xCC\xCC\x48\x8B\x47\x38",
+			"xxxxxxxxxxxxxxxx????xxxx"
```
Comment (hyperbx) [inline HedgeModManager/Resources/Codesv2/SonicFrontiers.hmm]: Fix indentation here, replace tabs with spaces for the mask string.

**Comment by wAABBsif** [comment]
Fixed the indentation, and also rewrote the description.

File: HedgeModManager/Resources/Codesv2/SonicFrontiers.hmm [hunk 2]
```diff
@@ -3085,6 +3085,21 @@ Code "Homing Attack Trick Animations" in "Animation" by "Hyper & Ahremic" does "
     }
 }
 
+Patch "Disable Quick Cyloop Target Override" in "Gameplay" by "WasifBoomz & The Duck Dealer" does "Disables the target change that under certain conditions prevents you from attacking an enemy after a Quick Cyloop."
```
Comment (hyperbx) [inline HedgeModManager/Resources/Codesv2/SonicFrontiers.hmm]: Please change the metadata to the following;
```
Patch "Fix Quick Cyloop Lock-On" in "Fixes" by "WasifBoomz & The Duck Dealer" does "Fixes the lock-on target changing whenever Sonic performs a Quick Cyloop, allowing for consistent attacks during combat."
```

**Pull request merged**